src/shared/copy/notifications/categories/accounts-users-orgs.ts

```diff
@@ -138,3 +138,9 @@ export const updateOrgFailed = (): Notification => ({
   message:
     'Error retrieving new organization information. Please refresh this page.',
 })
+
+export const updateQuartzOrganizationsFailed = (): Notification => ({
+  ...defaultErrorNotification,
+  message:
+    'Error retrieving organizations available to this account. Please refresh this page.',
```

Comment:
I think we can make this a little bit less mechanical sounding.

Reply:
Done.